### ISSUE TITLE
Unable to download Helm (Github API Rate limit)

### ISSUE THREAD

**Issue by chipironcin** [kubernetes/helm#2323]
Based on https://github.com/kubernetes/helm/blob/master/docs/install.md
When installing from script doing:

```
$ curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
$ chmod 700 get_helm.sh
$ ./get_helm.sh
```

I get:
>SHA sum of /tmp/helm does not match. Aborting.

After debugging I found the tag was emtpy inside the script.
After trying to get the tag manually:

```
$ curl https://api.github.com/repos/kubernetes/helm/releases/latest
{
  "message": "API rate limit exceeded for x.x.x.x. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)",
  "documentation_url": "https://developer.github.com/v3/#rate-limiting"
}
```

We should change the script to work by scrapping HTML content rather than using the Github API to avoid this issue.

**Comment by chipironcin** [comment]
Update: PR in its way

**Comment by technosophos** [comment]
Closed by #2324
